priv/repo/migrations/20171101183557_add_routing_key_to_subscription.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Aprb.Repo.Migrations.AddRoutingKeyToSubscription do
+  use Ecto.Migration
+
+  def change do
+    alter table(:subscriptions) do
+      add :routing_key, :string, default: "#"
```

Comment:
Seems like a surprising default to impose at the database layer. If these rows always get created via one code path, maybe it can just be imposed there?

(Or, are you setting it here so the column is never empty, even for freshly migrated rows? If so, should it be marked not-nullable?)

Reply:
good point i can remove that and assume `null` means listen to everything.